boards/arm/nucleo_f070rb/doc/nucleof070rb.rst

```diff
@@ -0,0 +1,179 @@
+.. _nucleo_f070rb_board:
+
+ST Nucleo F070RB
+################
+
+Overview
+********
+The STM32 Nucleo-64 development board with STM32F070RB MCU, supports Arduino and ST morpho connectivity.
+
+The STM32 Nucleo board provides an affordable, and flexible way for users to try out new concepts,
+and build prototypes with the STM32 microcontroller, choosing from the various
+combinations of performance, power consumption, and features.
+
+The Arduino* Uno V3 connectivity support and the ST morpho headers allow easy functionality
+expansion of the STM32 Nucleo open development platform with a wide choice of
+specialized shields.
+
+The STM32 Nucleo board integrates the ST-LINK/V2-1 debugger and programmer.
+
+The STM32 Nucleo board comes with the STM32 comprehensive software HAL library together
+with various packaged software examples.
+
+.. image:: img/nucleo_f070rb_board.jpg
+     :width: 500px
+     :height: 367px
+     :align: center
+     :alt: Nucleo F070RB
+
+More information about the board can be found at the `Nucleo F070RB website`_.
+
+Hardware
+********
+Nucleo F070RB provides the following hardware components:
+
+- STM32 microcontroller in QFP64 package
+- Two types of extension resources:
+    - Arduino* Uno V3 connectivity
```

Comment:
lists need a blank line before them, and should be indented to align with the preceding text:

    - Two types of extension resources:

      - Arduino ....

Notice the blank line before the (sub)list, and the ``-`` for the list item aligns with the ``T`` in the text of the item before.